plugins/lua/ts_lua_util.c

```diff
@@ -201,15 +201,15 @@ ts_lua_add_module(ts_lua_instance_conf *conf, ts_lua_main_ctx *arr, int n, int a
 
     if (conf->content) {
       if (luaL_loadstring(L, conf->content)) {
-        snprintf(errbuf, errbuf_size, "[%s] luaL_loadstring %s failed: %s", __FUNCTION__, conf->script, lua_tostring(L, -1));
+        snprintf(errbuf, errbuf_size, "[%s] luaL_loadstring '%s' failed: %s", __FUNCTION__, conf->content, lua_tostring(L, -1));
```

Comment:
i think it may be too much to put the entire script content here. 
Perhaps we can skip that in that ?